docs/5.management/1.account.md

```diff
@@ -0,0 +1,202 @@
+mandatory that they provide the corresponding justification why they
+need more resources. Exceptionally, if users need more resources, after
+a clear justification the FIWARE Lab nodes can provide extra resources
+to cover them.
+
+### Resource assignment and account upgrade by FIWARE Lab administrators
+
+This section describes how FIWARE Lab Node operators should process
+incoming requests. The process is simple and depicted below.
+
+Once, for example, a coach assigns you a ticket, you start progressing
+it, each comment will be notified to the end-user that will be in this
+way able to provide you further information. For resource
+assignment/negotiation the guide in Section [Quotas and Flavours]()
+should be considered.
+
+![Account upgrade workflow.](image15.png)
+
+If your node is not able to complete the assignment, we suggest
+negotiating the assignment of those resources in other FIWARE Lab node,
+taking into consideration user’s requirements and predefined
+accelerator's program to FIWARE Lab nodes mapping. Of course, you should
+inform users about this migration assignment to confirm that it is ok
+for them.
+
+#### How to assign resources to a new community user
+
+To manage user information, you have to use the Horizon-based Cloud
+Portal interface at
+[*https://cloud.lab.fiware.org*](https://cloud.lab.fiware.org). In this
+web site, you can to click on the button *Request Community Account* and
+you will see a dialog box requesting the information required to set up
+a community account. Once the user has filled all the information and
+submitted it, the process will start with the creation of the
+corresponding Jira ticket. The next steps that every FIWARE Lab node has
+to follow is the following:
+
+1.  Start progressing the ticket as soon as you receive it.
+
+1.  Check the requested resources and comment to inform the user about
+decision on resources, or additional information you need.
+
+1.  If the user is not created you have to create it.
+
+1.  If the user doesn’t have an associated project to create cloud
+resources, you have to create it and to assign the user as a member
+of the project.
+
+1.  Assign the role *Community* in the domain *default* to the user. The
+user can have several roles for different projects (owner, member,
+...), however, it is a Community or a Trial user for the
+whole domain.
+
+1.  Include in the metadata of the user (*description* field) the date
+in which the user has been upgraded to Community (with the format
+`"community_started_at": "YYYY-MM-DD"`).
+
+1.  Assign the desired quotas in your node to the user
+
+1.  Assign the endpoint group of your node to the user’s project.
+
+1.  If you need to contact the user you can use the email address
+included in the user detail.
+
+1. Progress to “done” when completed.
+
+1. Alternatively, if any impediment arises, move to impeded, and inform
+the user by commenting the ticket.
```

Comment:
docs/5.management/1.account.md: 155: MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "the user by commenting the tic..."]